examples/demo-app/src/cloud-providers/foursquare/foursquare-icon.js

```diff
@@ -1,3 +1,23 @@
+// Copyright (c) 2023 Uber Technologies, Inc.
```

Comment:
This file was contributed recently long after Uber stopped contributing and transferred ownership to OpenJS foundation. @igorDykhta @macrigiuseppe I assume it is an automated script that adds this, can we disable that?

Reply:
OpenJS Foundation council clarified we no longer need to carry forward the Uber copyright and instead use "Copyright contributors to the kepler.gl project" (without years)

Reply:
Opened #2467